commercial-partner.md

```diff
@@ -0,0 +1,52 @@
+# Commercial Partner
+
+**Note:** This document is a work in progress
+
+The Karmada commercial partner program offers Karmada support, consulting, professional services for organizations embarking on their Karmada journey.
+
+Karmada commercial partners are any organization that successfully leverages the Karmada project for their service offering. Such as:
+- Using Karmada to build service on a public or private cloud.
+- Repackaging Karmada as a core component of a service offering.
+- Offering Karmada technical support, consultation services, and training.
+- Others leverage Karmada in the manner it was intended to serve their customers.
+
+### Why become a partner?
+
+If your company provides professional services to support Karmada deployments, please consider applying to become a Karmada commercial partner.
+
+The main benefits of becoming a Karmada commercial partner are:
+* Placement on the first tab of https://karmada.io/
+* More community meeting exposure opportunities, such as the following meetings
+  * KubeCon.
+  * Online and offline meetings held by the Karmada community.
+  * Online and offline meetings participated by the Karmada community(such as KCD).
+* CNCF official media exposure opportunities, such as official Blog, Twitter, public accounts, etc.
```

Comment:
The description here may be a bit confusing, as you know the community should stay open and neutral, we should not use community channels to promote any commercial content, e.g. product pitch from any service provider.

Thus highlighting content neutrality would be helpful.
You may refer to "Marketing services and programs" section at https://www.cncf.io/services-for-projects/#marketing